changelog/unreleased/4435

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix the Shares screen not updating.
+
+Fixed the Shares screen not updating instantly when create/edit a link.
```

Comment:
And here I would write: 
`The items of the "Share" view are updated instantly when create/edit a link or share with users or groups in a non-root file.`